content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -99,10 +99,12 @@ includes the contents of all Secrets.
 
 ### Workload creation
 
-Users who are able to create workloads (either Pods, or
-[workload resources](/docs/concepts/workloads/controllers/) that manage Pods) will
-be able to gain access to the underlying node unless restrictions based on the Kubernetes
-[Pod Security Standards](/docs/concepts/security/pod-security-standards/) are in place.
+Permission to create workloads (either Pods, or
+[workload resources](/docs/concepts/workloads/controllers/) that manage Pods) in a namespace
+implicitly grants access to many other resources in that namespace, such as Secrets, ConfigMaps, and
+PersistentVolumes, which can be mounted into a Pod. Additionally, since Pods can run as any
```

Comment:
```suggestion
implicitly grants access to many other resources in that namespace that can be mounted in Pods, such as Secrets, ConfigMaps, and
PersistentVolumes. Additionally, since Pods can run as any
```